sdk/v2/system/api_client.go

```diff
@@ -48,3 +54,23 @@ func (a *apiClient) Ping(ctx context.Context) (PingResponse, error) {
 		},
 	)
 }
+
+func (a *apiClient) UnversionedPing(ctx context.Context) ([]byte, error) {
+	resp, err := a.SubmitRequest(
+		ctx,
+		rm.OutboundRequest{
+			Method:      http.MethodGet,
+			Path:        "ping",
+			SuccessCode: http.StatusOK,
+		},
+	)
+	if err != nil {
+		return []byte{}, errors.Wrap(err, "error submitting request")
+	}
+
+	respBodyBytes, err := ioutil.ReadAll(resp.Body)
```

Comment:
iirc, we should `defer resp.body.Close()`

Reply:
Oops, indeed; will add.